components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,276 +15,354 @@ gh_repo = "automate"
 
-- `with-user-sql` is required to handle user passwords and ensure user-specific association groups that are not duplicate.
+    For Example:
 
-- `--with-key-sql` is to handle cases where customers have users with multiple pem keys associated with their user or clients. The current chef-server API only dumps the default key. Sometimes, users will generate and assign additional keys to give additional users access to an account but still be able to lock them out later without removing everyone's access.
+    ```cmd
+        hab pkg exec chef/knife-ec-backup knife tidy server report --node-threshold 60 -s https://chef.io -u pivotal -k /etc/opscode/pivotal.pem
+    ```
 
-```cmd
-    hab pkg exec chef/knife-ec-backup knife ec backup backup_$(date '+%Y%m%d%H%M%s') --webui-key /etc/opscode/webui_priv.pem -s <chef server
-```
+4. Execute the below command to initiate a backup of your Chef Server data. 
 
-For example:
+    ```cmd
+        hab pkg exec chef/knife-ec-backup knife ec backup backup_$(date '+%Y%m%d%H%M%s') --webui-key /etc/opscode/webui_priv.pem -s <chef server url>
+    ```
+    In this command:
 
-```cmd
-    hab pkg exec chef/knife-ec-backup knife ec backup backup_$(date '+%Y%m%d%H%M%s') --webui-key /etc/opscode/webui_priv.pem -s https://chef.io`.
-```
+    - `--with-user-sql` is required to handle user passwords and ensure user-specific association groups are not duplicated.
+    - `--with-key-sql` is to handle cases where customers have users with multiple pem keys associated with their user or clients. The current chef-server API only dumps the default key. Sometimes, users will generate and assign additional keys to give additional users access to an account but still be able to lock them out later without removing everyone's access.
 
--  Execute the below command to clean unused data from reports. This is an optional steps
+    For example:
+    ```cmd
+        hab pkg exec chef/knife-ec-backup knife ec backup backup_$(date '+%Y%m%d%H%M%s') --webui-key /etc/opscode/webui_priv.pem -s https://chef.io`.
+    ```
 
-``` bash
-    hab pkg exec chef/knife-ec-backup knife tidy server clean --backup-path /path/to/an-ec-backup
-```
+    - Execute the below command to clean unused data from reports. This is an optional step
 
-5.   Execute the below command to copy the backup directory to the Automate HA Chef Server.
-```cmd
-     scp -i /path/to/key backup\_$(date '+%Y%m%d%H%M%s') user@host:/home/user
-```
+    ```bash
+        hab pkg exec chef/knife-ec-backup knife tidy server clean --backup-path /path/to/an-ec-backup
+    ```
 
-## Restore Backed Up Data to Chef Automate HA
+5. Execute the below command to copy the backup directory to the Automate HA Chef Server.
 
--   Execute the below command to install the habitat package for `knife-ec-backup`
+    ```cmd
+        scp -i /path/to/key <backup-file> user@host:/home/user
```

Comment:
backup file or path to backup file. Just like we are writing for key

Reply:
done.